src/main/scala/com/google/cloud/genomics/spark/examples/GenomicsConf.scala

```diff
@@ -58,13 +58,18 @@ class GenomicsConf(arguments: Seq[String]) extends ScallopConf(arguments) {
   }
 
   def getReferences = {
-    this.references().split(",").map(contig => {
-      val data = contig.split(":")
-      (data(0), (data(1).toLong, data(2).toLong))
-    }).toMap
+    Contig.parseContigsFromCommandLine(this.references())
   }
 }
 
 class PcaConf(arguments: Seq[String]) extends GenomicsConf(arguments) {
   val numPc = opt[Int](default = Some(2))
+  val allReferences = opt[Boolean]()
```

Comment:
Correct the, PcaConfig getReferences method actually excludes the X and Y contigs. Let me do make more explicit. On the general method they don't get excluded, also on the general method --all-references can't be specified.